### ISSUE TITLE
New to Coding - Issues Running Neural Style

### ISSUE THREAD

**Issue by epoworker** [jcjohnson/neural-style#467]
As the title suggests, I am using Neural style with no previous Python or general coding experience. I have been following [https://www.youtube.com/watch?v=3AnXiDksblA](this) tutorial to try and process video through Neural Style, but keep coming across errors. 

The code I have entered is as follows:
`for a in 'seq -w 000 100'; do
th neural_style.lua \
-style_image /home/ethanw/Downloads/ATTAMPTFOURR/PROCESS/Style/16684lpr_257494bc2fb728b.jpg \
-content_image /home/ethanw/Downloads/ATTAMPTFOURR/PROCESS/Light1/Light$a.tif \ 
-output_image /home/ethanw/Downloads/ATTAMPTFOURR/PROCESS/Output/Light1/Light_result_$a.jpg \
-gpu 0 \
-image_size 626 \
-style_weight 'shuf -i1100-1200 -n1' \
-backend nn \
-seed 100 \
-num_iterations 400 \
-content_weight 'shuf -i500-530 -n1' \
-init random \
-normalize_gradients ; done`
From this code, I get back:
`invalid argument: -w	
Usage: [options]
  -style_image          Style target image [examples/inputs/seated-nude.jpg]
  -style_blend_weights  [nil]
  -content_image        Content target image [examples/inputs/tubingen.jpg]
  -image_size           Maximum height / width of generated image [512]
  -gpu                  Zero-indexed ID of the GPU to use; for CPU mode set -gpu = -1 [0]
  -multigpu_strategy    Index of layers to split the network across GPUs []
  -content_weight       [5]
  -style_weight         [100]
  -tv_weight            [0.001]
  -num_iterations       [1000]
  -normalize_gradients  [false]
  -init                 random|image [random]
  -init_image           []
  -optimizer            lbfgs|adam [lbfgs]
  -learning_rate        [10]
  -lbfgs_num_correction [0]
  -print_iter           [50]
  -save_iter            [100]
  -output_image         [out.png]
  -style_scale          [1]
  -original_colors      [0]
  -pooling              max|avg [max]
  -proto_file           [models/VGG_ILSVRC_19_layers_deploy.prototxt]
  -model_file           [models/VGG_ILSVRC_19_layers.caffemodel]
  -backend              nn|cudnn|clnn [nn]
  -cudnn_autotune       [false]
  -seed                 [-1]
  -content_layers       layers for content [relu4_2]
  -style_layers         layers for style [relu1_1,relu2_1,relu3_1,relu4_1,relu5_1]
-output_image: command not found`

As far as I know, I have all the dependencies installed, and have tried running the code in both the default and `neural-style` directories.

Any help or clarification as to what I might be getting wrong would be much appreciated, thanks for bearing with me.

**Comment by AndreV84** [comment]
try with a simple piece of code starting with
th neural_style.lua
